src/styles.css

```diff
@@ -2,15 +2,52 @@
   font-family: sans-serif;
   text-align: center;
 }
-.homeview-user-names    {
-    font-family: Arial, Helvetica, sans-serif;
-    text-transform: capitalize;
+
+.homeview-user-names {
+  font-family: Arial, Helvetica, sans-serif;
+  text-transform: capitalize;
 }
 
-a:link, a:visited {
+a:link,
+a:visited {
   color: black;
   padding: 14px 25px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
 }
+
+.profile-view__background {
+  background-color: lightgrey;
+  background: url(https://img.lemde.fr/2019/04/22/0/191/1619/1079/688/0/60/0/e39da8d_2FIads9h8wB-0SwSgxVaVWsp.jpg)
+    lightgrey top center no-repeat;
+  background-size: 300px, contain;
+  margin-top: 70px;
+  margin-left: 100px;
+  display: flex;
+  justify-content: center;
+  width: 300px;
+  height: 400px;
+  position: center;
+  border-radius: 10px 10px;
+}
+
+.profile-view__image {
+  border-radius: 50%;
+  border: solid white 5px;
+  position: absolute;
+  width: 100px;
+  height: 100px;
+  margin-top: 100px;
+}
+
+.profile-view__title {
+  font-family: Helvetica;
+  text-transform: capitalize;
+  color: #2f4f4f;
+  margin-top: 250px;
```

Comment:
I think its too much

Reply:
Look at the image Carlos posted in issue #30